docs/graphql/manual/api-reference/schema-metadata-api/table-view.rst

```diff
@@ -4,7 +4,7 @@ Add or remove a table/view to Hasura GraphQL Engine's metadata using following A
 
 .. Note::
 
-   Only tables/views added to metadata are available for ``querying/mutating/subscribing`` data over **GraphQL** API.
+	Only tables/views added to metadata are available for ``querying/mutating/subscribing`` data over **GraphQL** API.
```

Comment:
What does this change achieve?

Reply:
@jamesgeorge007 Oh okay. That's cool. The change in `event-triggers.md` makes perfect sense :)

Reply:
@dsandip Hope everything is fine now :+1: